README.md

```diff
@@ -64,3 +64,15 @@
 ## React
 - [React Handbook by Faviocopes](https://www.freecodecamp.org/news/the-react-handbook-b71c27b0a795/) - Free fantastic e-book teaching you how to code with React.
 - [JavaScripts basics before learning React](https://dev.to/tracycss/the-vanilla-javascript-basics-to-know-before-learning-react-js-53aj) - Series of blog posts about JavaScript basics needed before learning React.
+
+## Technical Interview Study Material
```

Comment:
Please review https://github.com/jcsmileyjr/awesome-code-connector/pull/1 and let me know if you are opposed to those changes.